aiohttp/multipart.py

```diff
@@ -854,6 +856,8 @@ def _apply_content_transfer_encoding(self, stream):
         elif encoding == 'quoted-printable':
             for chunk in stream:
                 yield binascii.b2a_qp(chunk)
+        elif encoding == 'binary':
+            yield from stream
```

Comment:
Well, basically whole logic here is based around generators. Ok, will try to make a better test then to catch that problem. Thanks for report!